inc/tools.php

```diff
@@ -311,25 +336,21 @@ function cptui_get_single_taxonomy_registery( $taxonomy = [] ) {
 ?>
 
 	/**
-	 * Taxonomy: <?php echo esc_html( $taxonomy['label'] ); ?>.
-	 */
+	* Taxonomy: <?php echo esc_html( $taxonomy['label'] ); ?>.
+	*/
 
 	$labels = [
 		"name" => __( "<?php echo esc_html( $taxonomy['label'] ); ?>", "<?php echo esc_html( $textdomain ); ?>" ),
 		"singular_name" => __( "<?php echo esc_html( $taxonomy['singular_label'] ); ?>", "<?php echo esc_html( $textdomain ); ?>" ),
 <?php
-foreach ( $taxonomy['labels'] as $key => $label ) {
-	if ( ! empty( $label ) ) {
-		echo "\t\t" . '"' . esc_html( $key ) . '" => __( "' . esc_html( $label ) . '", "' . esc_html( $textdomain ) . '" ),' . "\n";
+	foreach ( $taxonomy['labels'] as $key => $label ) {
+		if ( ! empty( $label ) ) {
+			echo "\t\t" . '"' . esc_html( $key ) . '" => __( "' . esc_html( $label ) . '", "' . esc_html( $textdomain ) . '" ),' . "\n";
+		}
 	}
-}
 ?>
 	];
 
-	<?php
-	$show_graphql = isset( $taxonomy['show_in_graphql'] ) ? (bool) $taxonomy['show_in_graphql'] : false;
-	?>
```

Comment:
This looks like it was accidentally removed and we will want to have it in place still

Reply:
no, it's this comment above: "For the show_graphql, sort, and queryvar parts, the disp_boolean() function is immediately casting them to string, which is what we're getting from the settings extraction anyway, so may as well not worry about type casting there."  but I can leave it as it was,